src/components/Personality/PersonalityCard.jsx

```diff
@@ -33,12 +33,39 @@ class PersonalityCard extends Component {
                     <Row
                         style={{
                             padding: "10px 30px",
-                            marginTop: "10px",
-                            width: "100%"
+                            marginTop: "1em",
+
+                            width: "1000px",
+
+                            position: "relative",
+                            height: "156px",
+                            marginLeft: "-4.95%",
+                            marginRight: "-4%",
```

Comment:
We should avoid negative margins, if we need to do this, something is not right with the HTML structure.